app/jobs/submit_symphony_request_job.rb

```diff
@@ -3,14 +3,28 @@
 class SubmitSymphonyRequestJob < ActiveJob::Base
   queue_as :default
 
-  def perform(request, options = {})
+  def perform(request_id, options = {})
```

Comment:
I think that will work, but the author of Sidekiq seems to think otherwise:

https://github.com/mperham/sidekiq/wiki/Best-Practices#1-make-your-job-parameters-small-and-simple

It may only be a preference though:

http://stackoverflow.com/questions/30962979/should-you-still-pass-only-the-object-id-when-using-activejob